src/west/cmd/project.py

```diff
@@ -722,7 +724,17 @@ def _inf(project, msg):
     # Print '=== msg' (to clearly separate it from Git output). Supports the
     # same (foo) shorthands as the git commands.
 
-    log.inf('=== ' + _expand_shorthands(project, msg))
+    msg = '=== ' + _expand_shorthands(project, msg)
+
+    if platform.system() != "Windows" and os.isatty(sys.stdout.fileno()):
```

Comment:
This assumes that `log.inf()` will be going to stdout if stdout is a terminal. Not sure if that's an iffy assumption.

Reply:
Wouldn't it make more sense to put this in log itself? Maybe enabling color with a kwarg. That way other users can benefit as well.

Reply:
Sounds good. Will fix.

Reply:
Fixed